docs/catalog/ai/index.md

```diff
@@ -11,5 +11,4 @@ import Image from "@theme/IdealImage";
 
 <TeamMembers>
     <TeamMember github="navveenb">Navveen Balani</TeamMember>
-    <TeamMember github="tmcclell">Tammy McClellan</TeamMember>
```

Comment:
I'm not sure what happened here, but PRs should only push patterns and not changes to the website structure. Can you please clean this PR?